docs/user/checks.rst

```diff
@@ -372,59 +348,57 @@ Java MessageFormat
 Missing plurals
 ~~~~~~~~~~~~~~~
 
-Some plural forms are not translated. Check plural form definition to see for
-which counts each plural form is being used.
+Checks that plural forms are present in translation.
+Check plural form definition to see the specifics of how each plural form is being used.
```

Comment:
Not 100% sure what is going on here.

Reply:
This triggers in case the language has more plural forms, but the translation for some of them is empty.

Reply:
Thank you :)

Reply:
Is it possible to run the "Ellipsis" check on translated strings? It would make sense to move it up to the translation section, next to the "Trailing dots" check, which would also reduce some of the explanation, as it is shared between the two.

Reply:
The ellipsis check is for source strings - when souce uses three dots where ellipsis would be probably more appropriate.